### DISCUSSION TITLE
How to include node_modules for babel-loader?

### DISCUSSION THREAD

**Discussion by chkpnt** [webpack/webpack#16739] [Q&A]
I have a npm dependency, which needs to be transpiled by Babel. Unfortunately, just omitting `exclude: /node_modules/` has no effect. 

Here is a simple demonstrator, which requires Chart.js to be babeled for iOS 14: https://github.com/chkpnt/webpack-babel-loader-discussion16739

Otherwise, the application won't be working with iOS 14.0-14.4, as described here: https://github.com/chartjs/Chart.js/issues/11151

**Comment by alexander-akait** [comment]
Hello, you can use `[\\/]node_modules[\\/]((?!module).*)[\\/]`
